contracts/Vault.vy

```diff
@@ -1140,7 +1147,7 @@ def migrateStrategy(oldVersion: address, newVersion: address):
     assert self.strategies[newVersion].activation == 0
 
     strategy: StrategyParams = self.strategies[oldVersion]
-    self.strategies[oldVersion] = empty(StrategyParams)
+    self._revokeStrategy(oldVersion)
     self.strategies[newVersion] = strategy
```

Comment:
This is probably not the best way to handle this, probably want to individually set the values, namely the parameters (don't copy the gains/losses) and total debt

Reply:
your point is that we would consider it a completely new strat?

Reply:
The past performance of the old strategy and the future performance of the new strategy should be treated separately

Reply:
Did all the changes you requested but copied `totalDebt`.